copacetic/src/Components/MainPage/index.js

```diff
@@ -107,11 +107,12 @@ function MainPage() {
                 }      
             </div>
 
-            <div>
                 {showIcons && 
                 <Icons 
                 isDisplayed = {isDisplayed}
```

Comment:
indentation on everything being passed through - may need to be done on all of these